src/report.py

```diff
@@ -88,6 +87,8 @@ def download_report():
     :return: load test report in html format.
 
     """
+    if os.getenv('ROLE') != 'master': return
```

Comment:
why we need this if loop in the report?

Reply:
it's a condition, not loop ;) I'm not 100% sure, but it seems that the code may be executed on slaves as well as on master. This line should prevent it.